tests/virt/cluster/vm_cloning/test_vm_cloning.py

```diff
@@ -218,7 +218,7 @@ def test_clone_vm_with_instance_type_and_preference(
     ],
     indirect=True,
 )
-@pytest.mark.usefixtures("skip_on_psi_cluster")
+@pytest.mark.special_infra
 def test_clone_windows_vm(
     skip_if_workers_vms,
```

Comment:
also should be removed?
we have many places with this skip_if_workers_vms, seems they also not really actual anymore